pkg/k8s/portforward.go

```diff
@@ -177,6 +177,7 @@ func (pf *PortForward) Init() error {
 	go func() {
 		if err := pf.run(); err != nil {
 			failure <- err
+			return
 		}
 
 		select {
```

Comment:
I think we should remove this entire select block.  Regardless of how `pf.run()` returned, we want to do nothing since it is always the caller's responsibility to call `pf.Stop()`.  And then, since we're removing the select block, we can remove the early return too.

Reply:
:point_up: that makes a lot of sense to me! It is critical indeed to clarify the responsibility here.